src/display/api.js

```diff
@@ -69,6 +69,11 @@ const DefaultCMapReaderFactory =
     ? NodeCMapReaderFactory
     : DOMCMapReaderFactory;
 
+/**
+ * TODO https://github.com/mozilla/pdf.js/pull/10575
+ * @typedef {any} IPDFStream
```

Comment:
Also, since the `setPDFNetworkStreamFactory` neither is *nor should* be exposed in the API, is this actually necessary?
(The definition can be found in https://github.com/mozilla/pdf.js/blob/master/src/interfaces.js)

Reply:
I have no idea. I want to use pdf.js mainly for showing pdfs on screen and get the text that is stored in the pdf. But there is so much more you can do with this library that honestly, I have no idea. You tell me to remove it, it's gone. Somebody tells me it's useful, it stays.

All I want is to show pages of a PDF document ;)

So, should it stay or should it go?

Reply:
> Somebody tells me it's useful, it stays.

Not quite; as mentioned above, this simply shouldn't be part of the public API.

Reply:
removed